pages/api/v1/answers.ts

```diff
@@ -538,6 +533,44 @@ async function Angela( // Good ol' Angela
           throw new Error(`Action ${command.name} not found!`);
         }
 
+        // TODO: Do we need the new system messages? I.e. does the AI need to see if we've
+        // done a non ask_user correction?
+        const { newSystemMessages, corrections } =
+          await getCorrectionsForMissingCommandArgs(
+            chosenAction,
+            command,
+            chatGptPrompt.concat(newMessage) // This may contain useful information for the correction
```

Comment:
yeah cos you could have useful information in the system message right? like in the user description

Reply:
Using the original `chatGptPrompt` object seems a mistake to me since it includes the ~1000 token system message for how to be a Superflows chatbot.

This includes all the different functions it has available, the instructions on how to output `Reasoning`, `Plan`, `Tell user` and `Commands` etc.

I'd be surprised if it doesn't make performance worse since it adds a lot of tokens and gives instructions which conflict with what we want it to do